src/net_processing.cpp

```diff
@@ -23,6 +23,7 @@
 #include <random.h>
 #include <reverse_iterator.h>
 #include <scheduler.h>
+#include <timeoffsets.h>
```

Comment:
Are the following changes to Bitcoin unnecessary?

https://github.com/bitcoin/bitcoin/pull/29623/files#diff-6875de769e90cec84d2e8a9c1b962cdbcda44d870d42e4215827e599e11e90e3R1801

Reply:
'CNodeStateStats' has more information in Bitcoin than in Tapyrus. It was changed while refactoring for Peer manager class. In out code time offset is still in 'CNodeStats'